neat_internal.h

```diff
@@ -30,6 +30,14 @@
 
 #define NEAT_MAX_NUM_PROTO 4
 
+#if defined(USRSCTP_SUPPORT) || defined(HAVE_NETINET_SCTP_H)
+#define SCTP_SUPPORTED
```

Comment:
Why not be more specific as the code already is. I don't think we need this constant.